utils/uploadFile.ts

```diff
@@ -52,3 +52,12 @@ export function clearUploadFileData () {
     console.warn('Failed to clear sessionStorage:', error)
   }
 }
+
+export function fileToBase64 (file: File): Promise<string> {
+  return new Promise((resolve, reject) => {
+    const reader = new FileReader()
+    reader.readAsDataURL(file)
+    reader.onload = () => resolve(reader.result as string)
```

Comment:
The returned string is not a raw base64 string, and actually contains prefix like `data:image/png;base64`, which is a little bit confusing.